Client/src/components/Shared/ForgetPassword/forgetpassword.jsx

```
@@ -0,0 +1,203 @@
+  const emailId = useId();
+
+  // Load cooldown from cookie
+  useEffect(() => {
+    const loadCooldown = () => {
+      const cooldownData = document.cookie
+        .split('; ')
+        .find(row => row.startsWith('resetCooldown='));
+      
+      if (cooldownData) {
+        const [email, expiry] = cooldownData.split('=')[1].split(':');
+        const timeLeft = Math.ceil((parseInt(expiry) - Date.now()) / 1000);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)